src/main/java/com/meilisearch/sdk/Client.java

```diff
@@ -68,6 +68,37 @@ public Index createIndex(String uid, String primaryKey) throws Exception {
 		return meiliSearchIndexList;
 	}
 
+	
```

Comment:
```suggestion
```

Reply:
The new line is not necessary, but most of all, don't forget to use the editorconfig plugin to avoid this kind of useless whitespaces 😉